docs/source/configuration/development-mode.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: Development mode
+description: Running the router in development mode
+---
+
+During development, it is useful to run the Apollo Router with increased logging and relaxed security constraints.
+
+Development mode modifies the default configuration of the router to:
```

Comment:
The behavior of development mode doesn't really modify the default configuration. It sets/override the configuration. What I mean is if you disabled the sandbox explicitly in your config file, it will override that configuration to enable it. It's sometimes very annoying and TBH it bothered me since the beginning. For example this issue https://github.com/apollographql/router/issues/2404 is kinda related

Reply:
I've fixed this and hopefully simplified the code at the same time.